Funcky/GenericConstraints/RequireStruct.cs

```diff
@@ -0,0 +1,7 @@
+namespace Funcky.Constraints
```

Comment:
Namespace should be GenericConstraints

Reply:
review

Reply:
https://github.com/messerli-informatik-ag/funcky/pull/22/commits/e9963ad2bf1aa70235e144f8cdc8d5ac54bb6a10